catkin_tools/verbs/catkin_find/__init__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2015 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .cli import main
+from .cli import prepare_arguments
+
+# This describes this command to the loader
+description = dict(
+    verb='find',
+    description="Get the paths to important locations in a workspace.",
```

Comment:
Does this need to be update to include finding packages? I think it might work for both cases, though it is a little vague.